test/acceptance/kafo_configure_test.rb

```diff
@@ -68,6 +76,49 @@ module Kafo
       end
     end
 
+    describe 'verbose in the config file' do
+      it 'must respect verbose true' do
+        config = YAML.load_file(@config_file)
+        config[:verbose] = true
+        File.open(@config_file, "w") { |file| file.write(config.to_yaml) }
+
+        code, stdout, err = run_command '../bin/kafo-configure'
+
+        _(code).must_equal 0, err
+        _(stdout).must_include "INFO "
+        _(YAML.load_file(@config_file)[:verbose]).must_equal true
+      end
+
+      it 'must respect verbose false' do
+        config = YAML.load_file(@config_file)
+        config[:verbose] = false
+        File.open(@config_file, "w") { |file| file.write(config.to_yaml) }
+
+        code, stdout, err = run_command '../bin/kafo-configure'
+
+        _(code).must_equal 0, err
+        _(stdout).wont_include "INFO "
+        _(YAML.load_file(@config_file)[:verbose]).must_equal false
+      end
+
+      it 'must respect --verbose' do
+        code, stdout, err = run_command '../bin/kafo-configure --verbose'
+
+        _(code).must_equal 0, err
+        _(stdout).must_include "INFO "
+        _(YAML.load_file(@config_file)[:verbose]).must_equal true
+      end
+
+      it 'must respect --no-verbose' do
+        code, stdout, err = run_command '../bin/kafo-configure --no-verbose'
+        updated_config = YAML.load_file(@config_file)
+
+        _(code).must_equal 0, err
+        _(stdout).wont_include "INFO "
+        _(updated_config[:verbose]).must_equal false
+      end
```

Comment:
The `describe` block above suggests it is in the config file but the code looks like it isn't. This should be a new `describe` block, right?